src/systems/ec/implementations/vanilla/packages/ec-style-typography/ec-style-typography.scss

```diff
@@ -1,13 +1,31 @@
 /**
- * Typography rules
- * @define typography
- */
+* Typography styles
+* @define type
+*/
 
-// Import base and generic
+// Import base
 @import '@ecl/ec-base/ec-base';
-@import '@ecl/generic-style-typography/generic-style-typography';
 
-// Use generic mixin
+@mixin ecl-typography() {
+  /*
+   * Paragraph
+   */
+  .ecl-type--paragraph {
+    font: $ecl-font-prolonged-m;
+    max-width: 80ch;
```

Comment:
there's 1 issue with the `ch` unit: if you have both lead and normal paragraphs on your page, they won't have the same `max-width` (in pixels) -> designers probably won't like it. To be discussed with them

Reply:
Well in this case they should provide specifications for number of characters in each case.